interface.py

```diff
@@ -61,10 +61,13 @@ def return_cards(self):
         Return all cards of your program back to the dealt cards.
         """
         if not self.selection_confirmed:
-            for card in range(len(self.program)):
-                self.program[card] = None
-            self.cursor_index = 0
-            self.change_callback()
+            self.return_cards_for_power_down()
+
+    def return_cards_for_power_down(self):
```

Comment:
Zcela nechápu tohle rozdělení, nebo jsem nepochopila, jak má to power down fungovat. 
Chybí tady dokumentační řetezec a absolutně nechápu název co to má dělat když v těle metody není o power down ani slovo. Dále se mi nelíbí/nechápu, že když chci vracet karty(metoda return_card), tak volám vrátit karty pro power_down, jakože mi to nedává smysl. Nevím jestli je tady power down true nebo false.